### DISCUSSION TITLE
v1.0.0-beta8-fix2

### DISCUSSION THREAD

**Discussion by Mrs4s** [Mrs4s/go-cqhttp#1228] [General]
新增: 频道权限组管理相关api. by @Bluefissure 
新增: 获取频道帖子列表api. by @Mrs4s 
新增: 频道支持发送 `[CQ:video]` 了. by @wdvxdr1123 
新增: 模块化配置文件支持. by @fumiama 
新增: 动画表情 `[CQ:sticker]` 的支持. by @wdvxdr1123 

修复: 登录时可能出现 `unknown flag` 错误的问题. by @Mrs4s 
修复: 频道人数超过一定程度后无法解析消息的问题. by @Mrs4s 
修复: `HTTP POST` 没有 `Content-Type` 的问题. by @wasupandceacar 
修复: `Event` 连接仍然会监听 `API` 请求的问题. by @wdvxdr1123 
修复: `set_group_special_title` 无法正常使用的问题. by @wdvxdr1123 
修复: `websocket` 连接假死的问题. by @wdvxdr1123 

优化: 优化了程序的体积和内存占用. by @wdvxdr1123 
优化: 将频道相关的 `uint64` 类型字段转换为了 `string` 类型, 以方便某些语言处理. by @Mrs4s 

<hr /><em>This discussion was created from the release <a href='https://github.com/Mrs4s/go-cqhttp/releases/tag/v1.0.0-beta8-fix2'>v1.0.0-beta8-fix2</a>.</em>

**Comment by bklol** [comment]
win server2012 r2更新之后一直卡在connect to server
![QQ截图20211208193223](https://user-images.githubusercontent.com/54763946/145202433-c109633f-cad7-4e9d-ae0b-66ab2b4ff5fa.png)

**Comment by skymilong** [comment]
这段时间正好unknown flag ，更新后就好了

**Comment by dream-666** [comment]
为什么群聊不了，刚开始还能说一下

**Comment by chanchancl** [comment]
该版本碰到的一个crash
ubuntu amd64平台，http连接 android pad
```
panic: runtime error: makeslice: len out of range

goroutine 112 [running]:
github.com/Mrs4s/MiraiGo/binary.(*Reader).ReadBytes(...)
        github.com/Mrs4s/MiraiGo@v0.0.0-20211208080234-25c67a3ee1c1/binary/reader.go:40
github.com/Mrs4s/MiraiGo/internal/packets.parseSsoFrame({0xc0002720f3, 0xc00000eab8, 0xc0003cb4f0}, 0x1)
        github.com/Mrs4s/MiraiGo@v0.0.0-20211208080234-25c67a3ee1c1/internal/packets/global.go:145 +0x1a7
github.com/Mrs4s/MiraiGo/internal/packets.ParseIncomingPacket({0xc000144120, 0xc0003a4270, 0x10}, {0xc0003cb4f0, 0x10, 0x10})
        github.com/Mrs4s/MiraiGo@v0.0.0-20211208080234-25c67a3ee1c1/internal/packets/global.go:136 +0x38f
github.com/Mrs4s/MiraiGo/client.(*QQClient).netLoop(0xc0000b6480)
        github.com/Mrs4s/MiraiGo@v0.0.0-20211208080234-25c67a3ee1c1/client/network.go:296 +0x1c7
created by github.com/Mrs4s/MiraiGo/client.(*QQClient).connect.func1
        github.com/Mrs4s/MiraiGo@v0.0.0-20211208080234-25c67a3ee1c1/client/network.go:121 +0x307
```
